tests/test_config_reader.py

```diff
@@ -16,7 +16,8 @@ def test_parse_config_files():
         assert pytest_exit_object.value.code == 42  
         assert pytest_exit_object.sys.stdout.getline().strip() == "Directory containing config file, {}, not found. Exiting...".format("Not_Real_Directory")
     with pytest.raises(SystemExit) as pytest_exit_object:
-        config_data = parse_config_file(THIS_DIR)
+        os.makedirs("/Dummy_Directory")
+        config_data = parse_config_file("/Dummy_Directory")
         assert pytest_exit_object.type == SystemExit 
         assert pytest_exit_object.value.code == 42  
-        assert pytest_exit_object.sys.stdout.getline().strip() == "No file named config.yml found in config directory {}. Exiting...".format(THIS_DIR)
+        assert pytest_exit_object.sys.stdout.getline().strip() == "No file named config.yml found in config directory /Dummy_Directory. Exiting..."
```

Comment:
For this to pass you have to hard-code the actual directory path, not a dummy one.  But I change my mind on this as the name of the path will likely be different on windows and linux machines.  I think what you had originally, or else 
```
assert "No file named config.yml found in config directory" in pytest_exit_object.sys.stdout.getline().strip()
```
will do it.  Sorry about that!

Reply:
No problem, I reverted it back now.